src/bootstrap/suggest.rs

```diff
@@ -0,0 +1,142 @@
+use crate::builder::Builder;
+use glob::Pattern;
+use std::fmt::{self, Display};
+
+/// Suggests a list of possible `x.py` commands to run based on modified files in branch.
+pub fn suggest(build: &Builder<'_>, run: bool) {
+    // Find the commit the current branch split from master.
+    let merge_base = build
+        .config
+        .git()
+        .arg("merge-base")
+        .arg("HEAD")
+        .arg("origin/master")
```

Comment:
Please use `get_rust_lang_rust_remote` here instead of `origin`.

Hmm, running merge-base happens enough there should probably be a helper function for it; but this is fine for now.

Reply:
I can't find the function you are talking about? A lot of `src/bootstrap` seems to use hardcoded remotes.

Reply:
It was added in https://github.com/rust-lang/rust/pull/105702, you probably need to rebase over master. https://github.com/rust-lang/rust/pull/106286 is moving it so you might want to wait until that's merged, `origin` is fine in the meantime.

Reply:
Did you mean #105058 ?